docs/gethue/themes/stack-hue-theme/layouts/partials/home.en.html

```diff
@@ -174,4 +156,39 @@ <h2>Find and connect your data</h2>
       </div>
     </div>
   </section>
+  <section class="switchable feature-large switchable--switch bg--secondary">
+    <div class="container">
+      <div class="row justify-content-around">
+        <div class="col-md-6">
+          <img alt="Image" class="border--round box-shadow-wide" src="img/databases.svg" />
+        </div>
+        <div class="col-md-6 col-lg-5">
+          <div class="switchable__text input-append">
+            <h2>Get your own SQL assistant</h2>
```

Comment:
Need to be more explicit that this is about Slack and unfurling and link to install etc...